serving/samples/helloworld-java/README.md

```diff
@@ -190,7 +189,10 @@ folder) you're ready to build and deploy the sample app.
    xxxxxxx-ingressgateway   LoadBalancer   10.23.247.74   35.203.155.229   80:32380/TCP,443:32390/TCP,32400:32400/TCP   2d
    ```
 
-1. To find the URL for your service, use
+   **Note**
+   If your cluster is new, it may take sometime for the service to get asssigned an external IP address. Instead of rerunning the command, you can add `--watch` to the command below to view the component's status updates in real time. Use CTRL+C to exit watch mode. 
```

Comment:
```suggestion
   If your cluster is new, it may take some time for the service to get assigned an external IP address. Instead of rerunning the command, you can add `--watch` to the command below to view the component's status updates in real time. Use CTRL+C to exit watch mode. 
```

Reply:
@samodell Accepting your suggested change appears to have brought back the cla: no label. Will you please suggest how we could go about getting the cla: yes label back?